content/en/docs/languages/php/instrumentation.md

```diff
@@ -150,104 +290,308 @@ function as part of PHP's shutdown process:
-informational - these values will be emitted as part of the scope of any
-telemetry emitted by that tracer. All `Tracer`s that are created by a single
-`OpenTelemetry` instance will interoperate, regardless of differences in these
-parameters.
+The values of `instrumentation-scope-name` and `instrumentation-scope-version`
+should uniquely identify the
+[Instrumentation Scope](/docs/concepts/instrumentation-scope/), such as the
+package, module or class name. While the name is required, the version is still
+recommended despite being optional.
+
+It's generally recommended to call `getTracer` in your app when you need it
+rather than exporting the `tracer` instance to the rest of your app. This helps
+avoid trickier application load issues when other required dependencies are
+involved.
+
+In the case of the [example app](#example-app), there are two places where a
+tracer may be acquired with an appropriate Instrumentation Scope:
+
+First, in the _application file_ `index.php`:
+
+```php
+<?php
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LogLevel;
+use Slim\Factory\AppFactory;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use OpenTelemetry\API\Globals;
+
+require __DIR__ . '/vendor/autoload.php';
+
+require('dice.php');
+require('instrumentation.php');
+
+$tracerProvider = Globals::tracerProvider();
+$tracer = $tracerProvider->getTracer(
+  'dice-server',
+  '0.1.0',
+  'https://opentelemetry.io/schemas/1.24.0'
+);
+
+$logger = new Logger('dice-server');
+$logger->pushHandler(new StreamHandler('php://stdout', Logger::INFO));
+
+$app = AppFactory::create();
+
+$dice = new Dice();
+
+$app->get('/rolldice', function (Request $request, Response $response) use ($logger, $dice, $tracer) {
+// ...
+}
+```
+
+And second, in the _library file_ `dice.php`:
+
+```php
+<?php
+use OpenTelemetry\API\Globals;
+use OpenTelemetry\SemConv\TraceAttributes;
+
+class Dice {
+
+    private $tracer;
+
+    function __construct() {
+        $tracerProvider = Globals::tracerProvider();
+        $this->tracer = $tracerProvider->getTracer(
+          'dice-lib',
+          '0.1.0',
+          'https://opentelemetry.io/schemas/1.24.0'
+        );
+    }
+
+    public function roll($rolls) {
+        $result = [];
+        for ($i = 0; $i < $rolls; $i++) {
+            $result[] = $this->rollOnce();
+        }
+        return $result;
+    }
+
+    private function rollOnce() {
+      $result = random_int(1, 6);
+      return $result
+    }
+}
+```
 
 ### Create Spans
 
-To create [Spans](/docs/concepts/signals/traces/#spans), you only need to
-specify the name of the span. The start and end time of the span is
-automatically set by the OpenTelemetry SDK.
+Now that you have [tracers](/docs/concepts/signals/traces/#tracer) initialized,
+you can create [spans](/docs/concepts/signals/traces/#spans).
+
+The code below illustrates how to create an active span.
```

Comment:
I think "active span" should just refer to a span that has been activated via `$span->activate()`. So, remove the word "active"

Reply:
```suggestion
The code below illustrates how to create a span.
```